demos/dropbase.ftd

```diff
@@ -38,100 +38,208 @@ custom tools with SQL and Python in minutes.
 
 
 -- component header:
+optional boolean $open: false
```

Comment:
Meenu, why do you keep specifying a value and marking things optional. This is like 4th or 5th time we are telling you to not mark things optional if it has a value. 

NEVER write this:

```
-- component header:
optional boolean $open: false
```

Only this:

```
-- component header:
optional boolean $open: 
```

Which is something you probably rarely need, so never use `optional` understand you can yes, for this field I want it to have NO value. $open has a value, you set it to `false`, you always want `$open` to have a value, it must be either true or false.